### DISCUSSION TITLE
LvimIde in the Lazyman Neovim Configuration Manager

### DISCUSSION THREAD

**Discussion by doctorfree** [lvim-tech/lvim#62] [General]
I've added `LvimIde` to the supported Neovim configurations in the [Lazyman Neovim Configuration Manager](https://github.com/doctorfree/nvim-lazyman). `Lazyman` can be used to manage multiple Neovim configurations. The `LvimIde` configuration is installed and initialized with the command `lazyman -L LvimIde`.

The initialization process for `LvimIde` applies a patch to modify references to `.../nvim/...`, changing those to `.../nvim-LvimIde/...`. This is because the Lazyman Neovim configurations are installed in unique folders in `$HOME/.config` rather than the default Neovim locations. The environment variable `NVIM_APPNAME` is used to determine which config to use.

Note that this type of patching is not necessary if something like `vim.fn.stdpath("config")` is used rather than something like `home .. "/.config/nvim"`. But the patch works fine and the `LvimIde` config works well. It's an excellent Neovim configuration.

In addition to patching the hard coded paths where necessary, Lazyman modifies the  treesitter `ensure_installed` setting to install 10 basic parsers and adds the `auto_install = true` setting to install all others when their respective filetype is opened. This is done because  installing all treesitter parsers during initialization was time consuming and error prone.

You've really done an excellent job with the breadth and depth of coding language support and debugging facilities. Thanks!

Let me know if there are any changes you would like me to make with the incorporation of `LvimIde` in `Lazyman`.

**Comment by lvim-tech** [comment]
Thanks, @doctorfree
